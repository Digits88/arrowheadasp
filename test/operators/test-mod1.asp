Test Integer Modulus operator:
<hr>

19 Mod 6.7 = <%= 19 Mod 6.7 %><br>
5 mod 2 = <%= 5 Mod 2 %><br>
"5" mod 2 = <%= "5" Mod 2 %><br>
5 mod "2" = <%= 5 Mod "2" %><br>
"5" mod "2" = <%= "5" Mod "2" %><br>
IsNull(Null Mod 2) = <%= IsNull(Null Mod 2) %><br>
IsNull(1 Mod Null) = <%= IsNull(1 Mod Null) %><br>
IsNull(Null Mod Null) = <%= IsNull(Null Mod Null) %><br>
Empty Mod 2 = <%= Empty Mod 2 %><br>


