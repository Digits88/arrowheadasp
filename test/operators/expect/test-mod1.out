Test Integer Modulus operator:
<hr>

19 Mod 6.7 = 5<br>
5 mod 2 = 1<br>
"5" mod 2 = 1<br>
5 mod "2" = 1<br>
"5" mod "2" = 1<br>
IsNull(Null Mod 2) = True<br>
IsNull(1 Mod Null) = True<br>
IsNull(Null Mod Null) = True<br>
Empty Mod 2 = 0<br>


